test/unit/test.database.js

```diff
@@ -1,13 +1,247 @@
 const environments = require("../../knexfile.js");
 const Knex = require("knex");
 const { postProcessDatabaseResults } = require("src/server/post-process-database-results");
+const { processBatchOfLogs } = require("src/blockchain/download-augur-logs");
+const Augur = require("augur.js");
+const uuid = require("uuid");
+const deepmerge = require("deepmerge");
 
-module.exports = () => {
-  const env = Object.assign({}, environments.test, {
+
+function getEnv() {
+  return Object.assign({}, environments.test, {
     postProcessResponse: postProcessDatabaseResults,
   });
+}
+
+function contains(container, element) {
+  for (let i = 0; i < container.length; i++) {
+    if (container[i] === element) {
+      return true;
+    }
+  }
+  return false;
+}
+
+function required(args, ...params) {
+  args = Object.keys(args);
+  for (let p = 0; p < params.length; p++) {
+    const param = params[p];
+    if (!contains(args, param)) {
+      throw Error(`Required param "${param}" was not provided.`);
+    }
+  }
+}
+
+function seedDb(db) {
+  return Promise.resolve(null).then(async () => {
```

Comment:
Could this also be represented as a native async function without the `return Promise.resolve(null).then(` ?

Reply:
Yeah I can switch all of the promise stuff to async/await.